src/main/resources/config/liquibase/changelog/20250302174100_changelog.xml

```diff
@@ -0,0 +1,295 @@
+            <where>text LIKE '%/api/files%'</where>
+        </update>
+
+        <update tableName="exam">
+            <column name="confirmation_end_text" valueComputed="REPLACE(confirmation_end_text, '/api/files', '/api/core/files')"/>
+            <where>confirmation_end_text LIKE '%/api/files%'</where>
+        </update>
+
+        <update tableName="exam">
+            <column name="confirmation_start_text" valueComputed="REPLACE(confirmation_start_text, '/api/files', '/api/core/files')"/>
+            <where>confirmation_start_text LIKE '%/api/files%'</where>
+        </update>
+
+        <update tableName="exam">
+            <column name="end_text" valueComputed="REPLACE(end_text, '/api/files', '/api/core/files')"/>
+            <where>end_text LIKE '%/api/files%'</where>
+        </update>
+
+        <update tableName="exam">
+            <column name="start_text" valueComputed="REPLACE(start_text, '/api/files', '/api/core/files')"/>
+            <where>start_text LIKE '%/api/files%'</where>
+        </update>
+
+        <update tableName="exercise">
+            <column name="problem_statement" valueComputed="REPLACE(problem_statement, '/api/files', '/api/core/files')"/>
+            <where>problem_statement LIKE '%/api/files%'</where>
+        </update>
+
+        <update tableName="exercise_hint">
+            <column name="content" valueComputed="REPLACE(content, '/api/files', '/api/core/files')"/>
+            <where>content LIKE '%/api/files%'</where>
+        </update>
+
+        <update tableName="exercise_hint">
+            <column name="description" valueComputed="REPLACE(description, '/api/files', '/api/core/files')"/>
+            <where>description LIKE '%/api/files%'</where>
+        </update>
+
+        <update tableName="faq">
+            <column name="question_answer" valueComputed="REPLACE(question_answer, '/api/files', '/api/core/files')"/>
+            <where>question_answer LIKE '%/api/files%'</where>
+        </update>
+
+        <update tableName="feedback">
+            <column name="detail_text" valueComputed="REPLACE(detail_text, '/api/files', '/api/core/files')"/>
+            <where>detail_text LIKE '%/api/files%'</where>
+        </update>
+
+        <update tableName="feedback">
+            <column name="text" valueComputed="REPLACE(text, '/api/files', '/api/core/files')"/>
+            <where>text LIKE '%/api/files%'</where>
+        </update>
+
+        <update tableName="grading_instruction">
+            <column name="feedback" valueComputed="REPLACE(feedback, '/api/files', '/api/core/files')"/>
+            <where>feedback LIKE '%/api/files%'</where>
+        </update>
+
+        <update tableName="grading_instruction">
+            <column name="instruction_description" valueComputed="REPLACE(instruction_description, '/api/files', '/api/core/files')"/>
+            <where>instruction_description LIKE '%/api/files%'</where>
+        </update>
```

Comment:
Does feedback really allow Markdown? I thought is was a regular text box. If we do not need this migration, then this would remove one potentially big table that does not need to be touched.

Also applies to `long_feedback_text` further below.